image/templates/helm/shared/templates/_scanner-v4_init.tpl.htpl

```diff
@@ -163,7 +163,22 @@
 
 {{- end -}}
 
+{{- if eq $.Chart.Name "stackrox-secured-cluster-services" -}}
```

Comment:
I'd say we move this here, otherwise we would have a tight coupling between the logic in this file and the callers of this template.